src/legacy/core_plugins/visualizations/public/np_ready/public/embeddable/visualize_embeddable_factory.tsx

```diff
@@ -52,7 +53,9 @@ export class VisualizeEmbeddableFactory extends EmbeddableFactory<
 > {
   public readonly type = VISUALIZE_EMBEDDABLE_TYPE;
 
-  constructor() {
+  constructor(
+    private readonly getServices: () => Promise<Pick<VisualizationsStartDeps, 'uiActions'>>
```

Comment:
Done. I've changed it to `getUiActions`.

I've used `getServices` because it is convenient and analogous to what we have in `core.getStartServices`.